packages/sidebar-settings/src/blocks/dropdown.ts

```diff
@@ -22,6 +22,7 @@ export type DropdownBlock<AppBridge> = {
 
     /**
      * The size of the dropdown.
+     * @deprecated
```

Comment:
```suggestion
     * @deprecated `size` has no effect anymore, will be removed 13.09.2025 
```

Not sure about the date convention in here, check other deprecated functions within the brand sdk